docs/source/getting_started/v1_user_guide.md

```diff
@@ -0,0 +1,53 @@
+# vLLM V1 User Guide
+
```

Comment:
A missing item is that fallback for structured generation is not supported. So only `xgrammar:no_fallback` is supported

Reply:
added in the unsupported features section